SConstruct

```diff
@@ -592,6 +592,9 @@ if selected_platform in platform_list:
                 env.Append(CCFLAGS=["-g3"])
             else:
                 env.Append(CCFLAGS=["-g2"])
+            # Remap absolute paths to relative paths for debug symbols.
+            project_path = sys.path[0]
+            env.Append(CCFLAGS=[f"-ffile-prefix-map={project_path}=."])
```

Comment:
For the record I tried both with `=.` and just `=`, both work fine but there's no difference, relative paths still start with `./`.